lib/screens/play/card/card_actions/card_action_panel.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+
+import 'package:treasure_of_the_high_seas/model/card/action/card_action.dart'
+    as Model;
+import 'package:treasure_of_the_high_seas/model/card/action/card_action_details.dart';
+import 'package:treasure_of_the_high_seas/model/game_result.dart';
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+import './card_action_text.dart';
+import 'card_action_line.dart';
+
+class CardActionPanel extends StatelessWidget {
+  final Model.CardAction action;
+  final GameState state;
+  final bool readOnly;
+
+  CardActionPanel(this.action, this.state, { this.readOnly = false });
+
+  @override
+  Widget build(BuildContext context) {
+    final actionDetails = action?.actionDetails;
+    final enabled = action?.isEnabled(state) ?? false;
+
+    final shape = RoundedRectangleBorder(
+      side: BorderSide(color: Colors.black, width: 1),
+      borderRadius: BorderRadius.circular(10),
+    );
+
+    return Padding(
+        padding: EdgeInsets.symmetric(vertical: 4),
+        child: readOnly
+            ? Card(
+                shape: shape,
+                color: _getButtonColor(actionDetails),
+                child: _getActionComponents(actionDetails))
+            : RaisedButton(
```

Comment:
I was a little sad about this, but changing the widget type seemed to be the only way to prevent interaction properly. Mapping `onPressed` to a null function greys out the options, which I didn't want the scrying UI to do. Mapping it to a no-op means the actions still interact like buttons - they respond to being clicked (even though nothing happens). Meh.